indico/modules/events/registration/templates/display/registration_summary.html

```diff
@@ -97,11 +97,10 @@
                     </a>
                 </div>
             {% endif %}
-            {# TODO: check e-ticket #}
-            {% if true %}
+            {% if registration.state.name == 'complete' and regform.tickets_enabled
+                and regform.ticket_on_summary_page %}
```

Comment:
I think keeping this in one line is more readable. We don't have a strict length limit for lines in templates